pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -551,12 +551,18 @@ private static int GetWaitCycles(int millis = WAIT_TIME)
                 waitCycles *= Program.UnitTestTimeoutMultiplier;
             }
 
-            // Wait a little longer for debug build. (This may also imply code coverage testing, slower yet)
+            // Wait a little longer for debug build.
             if (ExtensionTestContext.IsDebugMode)
             {
                 waitCycles = waitCycles * 4;
```

Comment:
Make waitCycles *= 4;

Reply:
No problem. I just habitually don't change code that I don't have to, but agreed that consistency is good.